a couple blog posts (#36)
MAESTRO-Pool/website: content/blog/governance_catalyst/index.md

---
title: "Blockchain Governance and Project Catalyst"
description: "How Cardano will achieve full decentralization via blockchain governance"
date: 2021-10-12T12:47:56-07:00
lastmod: 2021-10-12T12:47:56-07:00
draft: false
weight: 10
contributors: [Varderes Barsegyan]
images: ["catalyst.jpeg"]
---

*Thomas Edison: The body is a community made up of its innumerable cells or inhabitants.*

## Blockchain Governance and Project Catalyst

> When it comes to blockchain technologies, Cardano sets itself apart in a completely unique and revolutionary way: **on-chain governance**. provides a mechanism for building, improving and maintaining a blockchain-based system in a truly decentralized and fully democratized way. In fact, solving this problem is Cardano's ultimate goal. It is the final "era" they have laid out in their roadmap: **[Voltaire](https://roadmap.cardano.org/en/voltaire/)**. Only after entry into this era can Cardano realize its vision of providing the world with a self-governing global financial operating system.

Reviewer: ```suggestion
When it comes to blockchain technologies, Cardano sets itself apart in a completely unique and revolutionary way: **on-chain governance**. On-chain governance provides a mechanism for building, improving and maintaining a blockchain-based system in a truly decentralized and fully democratized way. In fact, solving this problem is Cardano's ultimate goal. It is the final "era" they have laid out in their roadmap: **[Voltaire](https://roadmap.cardano.org/en/voltaire/)**. Only after entry into this era can Cardano realize its vision of providing the world with a self-governing global financial operating system.
```

> When it comes to blockchain technologies, Cardano sets itself apart in a completely unique and revolutionary way: **on-chain governance**. provides a mechanism for building, improving and maintaining a blockchain-based system in a truly decentralized and fully democratized way. In fact, solving this problem is Cardano's ultimate goal. It is the final "era" they have laid out in their roadmap: **[Voltaire](https://roadmap.cardano.org/en/voltaire/)**. Only after entry into this era can Cardano realize its vision of providing the world with a self-governing global financial operating system.

Reviewer 2: way: **on-chain governance**. provides --> Cardano's governance provides

Solving governance means solving an incredibly complex set of problems that require both high-level, abstract ideas and feasible, real-life engineering. Research in this field is a beautiful amalgamation of political science, economics, computer science, and game theory. Infact, humanity has been experimenting with solutions to these problems since the dawn of civilization.

![civilization](civ.png)
<center>Source: ancient-origins.net</center>

<br>

> Our ancestors have invented Democracy, Monarchy, Communism, Oligarchy, Theocracy, Dictatorship, and everything in between. Though these forms of government have been effective-at least temporarily-in their particular contexts, Charles Hoskinson, the Cardano team, and members of the community have taken it upon themselves to establish a newer form of government, one that is fair, incorruptible, and future-proof.

Reviewer 2: +1 on either parentheses or even a comma!

> 

Reviewer 2: ```suggestion
<br>
```
Need a line break here or an additional return

* **Assessment**: final review from Community Advisors
* **Voter discussion**: determine which proposals will be funded and discuss details with proposers

During Governance, community members vote and the winners are determined:
* **Registration**: voters register their Yoroi or Daedalus wallets with the Project Catalyst mobile app; registered wallets must contain at least 500 ADA to be eligible for voting
* **Voting**: voters vote on proposals; the weight of each person's vote depends on how much ADA they hold in their wallet
* **Tallying**: a voting committee tallies votes and publishes the results to the community

Finally, during Execution, rewards are distributed to the winners, which in turn start to implement their proposals:
* **Rewarding**: rewards are distributed to approved proposers, voters and Community Advisors
* **Reporting**: approved voters send progress reports for the next 3 months

The status of the current cycle can be seen in this highly informative [gantt chart](https://cardanocataly.st/en/funds/#)! Here is the breakdown of Fund 6:

![F6 gantt](gantt.png)
<center>Source: cardanocataly.st</center>

## Incentivizing participation

> Project Catalyst grows significantly at every funding cycle. This is no accident. The organizers of Catalyst, led by Dor Garbash[iohk team link], have established highly effective incentives that enable members of the community to actively participate. If you're a **proposer**, you can potentially receive funding to work on a project you are passionate about and that the Cardano community finds valuable. As a **voter**, you receive ADA for simply voting on your favorite proposals. Finally, as a **Community Advisor** (or Veteran Community Advisor you have been advising since a previous cycle), you are rewarded ADA for providing constructive feedback to proposals and assessing them based on *Impact*, *Auditability* and *Feasibility*. 

Reviewer 2: ```suggestion
Project Catalyst grows significantly at every funding cycle. This is no accident. The organizers of Catalyst, led by [Dor Garbash](https://iohk.io/en/team/dor-garbash), have established highly effective incentives that enable members of the community to actively participate. If you're a **proposer**, you can potentially receive funding to work on a project you are passionate about and that the Cardano community finds valuable. As a **voter**, you receive ADA for simply voting on your favorite proposals. Finally, as a **Community Advisor** (or Veteran Community Advisor if you have been advising since a previous cycle), you are rewarded ADA for providing constructive feedback to proposals and assessing them based on *Impact*, *Auditability* and *Feasibility*. 
```

The size of these rewards depends on the current funding cycle. For example, a total of *$4 million* worth of ADA was distributed to proposers, voters and community advisors for *Fund 6*!

## How to get involved
* **Writing proposals**: Proposals can be written in **[https://cardano.ideascale.com/](https://cardano.ideascale.com/)** during the Execution phase described above

> Community advisement: In addition to writing proposals, you can provide comments to initial proposals and assess them after they have been finalized. Again, this is done during the Execution phase.

Reviewer 2: ```suggestion
* **Community advisement**: In addition to writing proposals, you can provide comments to initial proposals and assess them after they have been finalized. Again, this is done during the Execution phase.
```

> Community advisement: In addition to writing proposals, you can provide comments to initial proposals and assess them after they have been finalized. Again, this is done during the Execution phase.

Reviewer: ```suggestion
**Community advisement**: In addition to writing proposals, you can provide comments to initial proposals and assess them after they have been finalized. Again, this is done during the Execution phase.
```

* **Catalyst Swarm**: Swarm is dedicated to organizing meetings, sessions and events to connect people and projects within the Catalyst ecosystem. They have dedicated meetings for deep diving into specific funding categories and proposals that have started to gain a notable amount of traction.
* **Discord**
  - [Cardano Project Catalyst](https://discord.gg/DKkteJWp)
  - [Catalyst Swarm](https://discord.gg/9pbRF2PfcU)
  - [Cardano Community](https://discord.gg/b3FVumGd)
  - [IOG Technical Community](https://discord.gg/inputoutput)
* **Telegram**
  - [@cardanocatalyst](https://t.me/cardanocatalyst)
  - [@ProjectCatalystChat](https://t.me/ProjectCatalystChat)
  - [@projectcatalystproposers](https://t.me/catalystproposers)
  - [@CatalystCommunityAdvisors](https://t.me/CatalystCommunityAdvisors)

## Resources

> * [https://projectcatalyst.org](https://projectcatalyst.org): Here you can find an introduction to Project Catalyst, a Glossary, a live calendar, and links to other relevant resources!

Reviewer 2: ```suggestion
* [https://projectcatalyst.org](https://projectcatalyst.org): Here you can find an introduction to Project Catalyst, a glossary, a live calendar, and links to other relevant resources!
```

* [https://cardano.ideascale.com](https://cardano.ideascale.com): The place where you can actually submit proposals and provide feedback to other proposers.
* [https://cardanocataly.st](https://cardanocataly.s): This is a community built collection of resources to help curious newcomers navigate the Catalyst ecosystem. It was funded via a Catalyst proposal!
* [https://cardanocataly.st/en/funds](https://cardanocataly.st/en/funds): A helpful chart to track the status of fund cycles
* [FAQs](https://docs.google.com/document/d/1qYtV15WXeM_AQYvISzr0a0Qj2IzW3hDvhMBvZZ4w2jE/edit#heading=h.i9248kyoe6de)

## The endgame of Catalyst

> Project Catalyst has been an incredible success for Cardano, in particular, and for blockchain, in general. There are now over 50,000 wallets registered to vote and the number of proposals are growing exponentially at every fund cycle. Not only is it a proven system that helps kick start projects within the ecosystem, the system itself is set up to evolve and adapt to the world and the needs of humanity. Eventually, Catalyst will evolve into a fully self-sustaining, digitized government that other countries and organizations around the world can use as their underlying governance infrastructure. The trajectory of this project points to a future where the **Voltaire** era is realized and Cardano's ultimate mission is achieved!

Reviewer 2: ```suggestion
Project Catalyst has been an incredible success for Cardano, in particular, and for blockchain in general. There are now over 50,000 wallets registered to vote and the number of proposals are growing exponentially at every fund cycle. Not only is it a proven system that helps kick start projects within the ecosystem, the system itself is set up to evolve and adapt to the world and the needs of humanity. Eventually, Catalyst will evolve into a fully self-sustaining, digitized government that other countries and organizations around the world can use as their underlying governance infrastructure. The trajectory of this project points to a future where the **Voltaire** era is realized and Cardano's ultimate mission is achieved!
```

> Project Catalyst has been an incredible success for Cardano, in particular, and for blockchain, in general. There are now over 50,000 wallets registered to vote and the number of proposals are growing exponentially at every fund cycle. Not only is it a proven system that helps kick start projects within the ecosystem, the system itself is set up to evolve and adapt to the world and the needs of humanity. Eventually, Catalyst will evolve into a fully self-sustaining, digitized government that other countries and organizations around the world can use as their underlying governance infrastructure. The trajectory of this project points to a future where the **Voltaire** era is realized and Cardano's ultimate mission is achieved!

Reviewer: ```suggestion
Project Catalyst has been an incredible success for Cardano, in particular, and for blockchain in general. There are now over 50,000 wallets registered to vote and the number of proposals are growing exponentially at every fund cycle. Not only is it a proven system that helps kick start projects within the ecosystem, the system itself is set up to evolve and adapt to the world and the needs of humanity. Eventually, Catalyst will evolve into a fully self-sustaining, digitized government that other countries and organizations around the world can use as their underlying governance infrastructure. The trajectory of this project points to a future where the **[Voltaire](https://roadmap.cardano.org/en/voltaire/)** era is realized and Cardano's ultimate mission is achieved!
```
